swift_scality_backend/diskfile.py

```diff
@@ -481,6 +482,16 @@ def open(self):
         if metadata is None:
             raise swift.common.exceptions.DiskFileDeleted()
         self._metadata = metadata or {}
+
+        try:
+            x_delete_at = int(self._metadata['X-Delete-At'])
+        except KeyError:
+            pass
+        else:
```

Comment:
If I don't catch ValueError, then the exception will bubble up and won't be ignored... Right ?